edx_solutions_api_integration/urls.py

```diff
@@ -30,7 +30,6 @@
     url(r'^organizations/*', include('edx_solutions_organizations.urls')),
     url(r'^mobile/v1/', include('edx_solutions_api_integration.mobile_api.urls')),
     url(r'^imports/*', include('edx_solutions_api_integration.imports.urls')),
-    url(r'^courses_metadata/', include('course_metadata.urls')),
```

Comment:
I could not see this API being used by Apros.

Reply:
Is this connected to the old progress code?  I'm not familiar with the CourseAggregatedMetadata model or this api.  Do you know what it was intended to do?

Reply:
I've added it back since it wasn't really in scope. I was just cleaning up some unused code while I was at it, but I don't want to blow up the scope and review time too much here.